.github/workflows/coveralls.yaml

```diff
@@ -5,13 +5,13 @@ on: [push, pull_request]
 jobs:
   gen_coverage:
     name: Calculate test coverage
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
```

Comment:
I tried to update to ubuntu-23-10, but [that is apparently not supported](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners/about-github-hosted-runners#standard-github-hosted-runners-for-public-repositories)